manager/state/raft/storage/walwrap_test.go

```diff
@@ -231,12 +231,24 @@ func TestReadRepairWAL(t *testing.T) {
 	require.NoError(t, err)
 	require.Len(t, files, 1)
 
-	fName := filepath.Join(tempdir, files[0].Name())
-	fileContents, err := os.ReadFile(fName)
-	require.NoError(t, err)
-	info, err := files[0].Info()
-	require.NoError(t, err)
-	require.NoError(t, os.WriteFile(fName, fileContents[:200], info.Mode()))
```

Comment:
Replaced this with `os.Truncate()` below, which was shorter, and more descriptive what we're doing 😅